test/runtime/monitor.go

```diff
@@ -36,6 +36,10 @@ const (
 	// value for the Cilium monitor
 	MonitorTraceNotification = "TraceNotification"
 
+	// MonitorEarlyTrace represents the EarlyTraceNotification configuration
```

Comment:
Nit: this definition is somewhat circular; i.e., what differentiates an early trace notification vs. a trace notification? It's not obvious without looking at the code itself.

Reply:
Yeah, this comment doesn't provide much helpful context but it's in line with each of the comments above the other configuration options declared in this file. Maybe we can do better at the actual option definition location.

Reply:
I got the same feeling  when I read it as a user in `Enable early trace when packets are received from endpoint`